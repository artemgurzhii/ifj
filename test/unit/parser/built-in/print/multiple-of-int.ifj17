print 42; 24; 424;
