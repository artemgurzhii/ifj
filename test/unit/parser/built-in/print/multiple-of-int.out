(print
  (int 42)
  (int 24)
  (int 424))
